templates/mdevents.conf.j2

```diff
@@ -0,0 +1,22 @@
+{% if collectd_plugin_mdevents_interval is not defined %}
+LoadPlugin mdevents
+{% else %}
+<LoadPlugin mdevents>
+   Interval {{ collectd_plugin_mdevents_interval }}
+</LoadPlugin>
+{% endif %}
+
+<Plugin "mdevents">
+{% if collectd_plugin_mdevents_event is defined %}
+  Event "{{ collectd_plugin_mdevents_event }}"
+{% endif %}
+{% if collectd_plugin_mdevents_ignore_event is defined %}
+  IgnoreEvent {{ collectd_plugin_mdevents_ignore_event | string | lower }}
```

Comment:
shouldn't these event being encapsulated as strings in ticks? "" missing?

Reply:
IgnoreArray is a boolean, which doesn't need to be encapsulated as a string